CRM/Contribute/Form/ManagePremiums.php

```diff
@@ -103,7 +133,7 @@ public function buildQuickForm() {
     $this->addElement('text', 'imageUrl', ts('Image URL'));
     $this->addElement('text', 'thumbnailUrl', ts('Thumbnail URL'));
 
-    $this->add('file', 'uploadFile', ts('Image File Name'), ['onChange' => 'select_option();']);
+    $this->add('file', 'uploadFile', ts('Image File Name'), ['onChange' => 'CRM.$("input[name=imageOption][value=image]").prop("checked", true);']);
```

Comment:
This is a bit offtopic, but the `select_option`  function did not exist. I looked as far back as CiviCRM 4.2 and could not find it. The jQuery snippet is ugly, but avoids having to add a new function just for that.